test/scripts/agent-images/prepare_agent_config.sh

```diff
@@ -1,25 +1,6 @@
 #!/usr/bin/env bash
 
-SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
-$("${SCRIPT_DIR}/../get_endpoints.sh")
-
-OUTFILE=bin/agent/etc/flightctl/config.yaml
 mkdir -p bin/agent/etc/flightctl/certs
-cp ~/.flightctl/certs/ca.crt bin/agent/etc/flightctl/certs/ca.crt
-cp ~/.flightctl/certs/client-enrollment.{crt,key} bin/agent/etc/flightctl/certs/
 
-echo "Creating agent config in ${OUTFILE}"
-tee ${OUTFILE} <<EOF
-enrollment-service:
-  authentication:
-    client-certificate: certs/client-enrollment.crt
-    client-key: certs/client-enrollment.key
-  service:
-    certificate-authority: certs/ca.crt
-    server: ${FLIGHTCTL_API_ENDPOINT}
-  enrollment-ui-endpoint: ${FLIGHTCTL_UI_ENDPOINT}
-grpc-management-endpoint: ${FLIGHTCTL_AGENT_GRPC}
-spec-fetch-interval: 0m10s
-status-update-interval: 0m10s
-tpm-path: /dev/tpmrm0
-EOF
+echo Requesting enrollment enrollment certificate/key and config for agent =====
+./bin/flightctl certificate request -n client-enrollment -d bin/agent/etc/flightctl/certs/ | tee bin/agent/etc/flightctl/config.yaml
```

Comment:
```suggestion
./bin/flightctl certificate request -d bin/agent/etc/flightctl/certs/ | tee bin/agent/etc/flightctl/config.yaml

# enforce the agent to fetch the spec and update status every 2 seconds to improve the E2E test speed
cat <<EOF | tee -a  bin/agent/etc/flightctl/config.yaml
spec-fetch-interval: 0m2s
status-update-interval: 0m2s
EOF
```

Reply:
We were missing a couple of settings which are important for E2E testing, the update intervals, otherwise the tests need to wait too long... this is why the console timeout was materializing, the agent takes too long to realize there is a console to be served... the test gives up.

Also note that I have removed the -n client-enrollment

The reason for that, is that if you run this a second time, it will still work because of the random name, otherwise it fails, and you can't run make e2e-test  twice without throwing again and redeploying or deleting csr/client-enrollment.

Reply:
Interesting, these fields were removed deliberately in https://github.com/flightctl/flightctl/pull/522/ 

That sounds good, I had just put `-n client-enrollment` to make it match `prepare_agent_config.sh` as I was not sure if the name is important in terms of where the file is placed.

Reply:
I see the intervals are needed just for testing.